Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView/ConsentTransparencyView.swift

```diff
@@ -81,7 +82,11 @@ public final class ConsentTransparencyView: UIView {
 
     public var detailText: String = "" {
         didSet {
-            detailLabel.text = detailText
+            let paragraphStyle = NSMutableParagraphStyle()
+            paragraphStyle.lineSpacing = 3
+            let attributedString = NSMutableAttributedString(string: detailText)
+            attributedString.addAttribute(.paragraphStyle, value: paragraphStyle, range: NSMakeRange(0, attributedString.length))
```

Comment:
This stuff makes the text look a lot prettier. I didn't even think about doing this. Nice! 👌